### PR TITLE
fix: Address update

### PR DESCRIPTION
**Pull request by ununhexium**
https://github.com/sovity/edc-ce/issues/1018

```[tasklist]
### Checklist
- [x] The PR title is short and expressive.
- [x] I have updated the CHANGELOG.md. See [changelog_update.md](https://github.com/sovity/authority-portal/tree/main/docs/dev/changelog_updates.md) for more information.
- [ ] I have updated the Deployment Migration Notes Section in the CHANGELOG.md for any configuration / external API changes.
- [ ] I have updated the Community Edition [Postman-Collection](https://github.com/sovity/edc-ce/blob/main/docs/api/postman_collection.json) if I changed existing APIs or added new APIs (e.g. for Management-API or API-Wrapper)
- [x] I have performed a **self-review**
```

### CODE REVIEW

File: tests/src/test/java/de/sovity/edc/e2e/UiApiWrapperTest.java [hunk 1]
```diff
@@ -591,6 +594,7 @@ void checkIdAvailability(E2eScenario scenario, @Provider EdcClient providerClien
         assertThat(posContractDefinitionResponse.getAvailable()).isTrue();
     }
 
+    @DisabledOnGithub
```
Comment (kamilczaja) [inline tests/src/test/java/de/sovity/edc/e2e/UiApiWrapperTest.java:597]: 💀

**Review by kamilczaja — CHANGES_REQUESTED** [review]
Changelog missing

File: tests/src/test/java/de/sovity/edc/e2e/UiApiWrapperTest.java (hunk 1 above)
Comment (ununhexium) [reply tests/src/test/java/de/sovity/edc/e2e/UiApiWrapperTest.java:597]: I still have a ticket open for this.

Comment (ununhexium) [reply tests/src/test/java/de/sovity/edc/e2e/UiApiWrapperTest.java:597]: https://github.com/sovity/edc-ce/issues/870

**Review by kamilczaja — APPROVED** [review]

**Pull request merged**